### PR TITLE
:feature: Feature/#649 contact section

### PR DESCRIPTION
**Pull request by ciaranschutte**

### CODE REVIEW

**Comment by ciaranschutte** [comment]
Super basic formatting for the phone number but consistent formatting and validation for all countries is beyond the scope of this ticket

File: src/common/displayFormatters.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const formatPhoneNumber = phone => {
```
Comment (ciaranschutte) [inline src/common/displayFormatters.js]: Super basic formatting for the phone number but consistent formatting and validation for all countries is beyond the scope of this ticket

**Comment by bdolly** [comment]
need to make sure we have analytics plugged into this before it rolls out to prod

**Comment by hlminh2000** [comment]
LGTM. @bdolly did you wanna add tracking to this before we merge? otherwise I'm ok for merging this.

**Comment by ciaranschutte** [comment]
@bdolly @hlminh2000  I'm going to merge this and create a ticket for analytics

**Pull request merged**